src/PrestaShopBundle/Form/Admin/Sell/Address/CustomerAddressType.php

```diff
@@ -106,47 +123,41 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ]),
                     new ExistingCustomerEmail(),
                 ],
+                'attr' => [
+                    'data-customer-information-url' => $this->router->generate('admin_customer_for_address_information'),
+                ],
             ]);
         } else {
             $builder->add('id_customer', HiddenType::class);
         }
 
-        $builder->add('phone_mobile', TextType::class, [
+        $builder->add('dni', TextType::class, [
+            'label' => $this->translator->trans('Identification number', [], 'Admin.Orderscustomers.Feature'),
+            'help' => $this->translator->trans(
+                'The national ID card number of this person, or a unique tax identification number.',
+                [],
+                'Admin.Orderscustomers.Feature'
```

Comment:
Also localized in _Admin.Orderscustomers.Help_, I let @Junebyun decide what domain she wants to keep. ;-)

Reply:
I'd rather keep _Admin.Orderscustomers.Feature_ because most strings of this PR are located in this domain!